python/tests/test_types.py

```diff
@@ -56,6 +56,7 @@
 
 @pytest.mark.parametrize("input_index, input_type", non_parameterized_types)
 def test_repr_primitive_types(input_index, input_type):
+    assert input_type.is_primitive
```

Comment:
It doesn't look like these tests are related to this PR. Can you move these to a separate PR?

Reply:
Sure, removed.